server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -0,0 +1,159 @@
+package com.google.musicanalysis.site;
+
+import com.google.gson.*;
+import com.google.musicanalysis.api.naturallanguage.*;
+import com.google.musicanalysis.api.perspective.*;
+import com.google.musicanalysis.api.youtube.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/analysis")
+public class AnalysisServlet extends HttpServlet {
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse res)
+      throws ServletException, IOException {
+
+    res.getWriter().write("<h1>Comment Analysis</h1>\n");
+
+    String videoName = "seun omonije";
+
+    // Use like this: {url_parameter, value}
+    HashMap<String, String> args = new HashMap<>();
+
+    args.put("q", videoName);
+    String videoIdJson = new YoutubeRequest("search", args).getResult();
+    String videoId = getVideoId(videoIdJson);
+
+    args.remove("q");
```

Comment:
Make a new variable instead of re-using a generic one. You can then name the previous one videoArgs and this one commentsArgs. This will make your code more readable and easier to grow.